packages/edit-post/undocumented-exports.md

```diff
@@ -0,0 +1,4 @@
+# Undocumented exports
+
+initializeEditor
```

Comment:
What's the purpose of this file? Does it mean we don't want to have them exposed? Is there JSDoc syntax like `@private` which would reflect it?

Reply:
A leftover of a test that I committed by accident. This makes no sense and I've deleted it.